{"title":"ChatGPT ล่มเช้านี้ ต้อนรับ Apple Intelligence รองรับการเชื่อมต่อวันแรก","link":"https://www.blognone.com/node/143597","date":1733963636000,"content":"<div><div><div><p>OpenAI ยืนยันปัญหาของบริการหลักทั้ง ChatGPT, Sora และ API ของนักพัฒนา ไม่สามารถใช้งาน ตั้งแต่เวลา 6:17น. ตามเวลาในไทย</p>\n<p>ข้อมูลจาก <a href=\"https://downdetector.com/status/openai/\">Downdetector</a> ยืนยันปัญหานี้เช่นกันในช่วงเวลาประมาณ 6:10น. และยังคงมีปัญหาอยู่</p>\n<p>อัปเดตล่าสุดของ OpenAI ในเวลา 7:24น. บอกว่าระบุสาเหตุของปัญหาได้แล้ว และกำลังแก้ไขให้เป็นปกติโดยเร็วที่สุด</p>\n<p>ในวันนี้ OpenAI ได้ประกาศของใหม่ตามแคมเปญ <a href=\"https://www.blognone.com/node/143462\">12 วัน 12 อย่าง</a> นั่นคือ <a href=\"https://www.blognone.com/node/143588\">Apple Intelligence รองรับการเชื่อมต่อกับ ChatGPT</a> ได้แล้ว จึงถือเป็นการเปิดตัวแบบยิ่งใหญ่จริง</p>\n<p>ที่มา: <a href=\"https://x.com/OpenAI/status/1867000372826607627\">OpenAI</a></p>\n<p><img alt=\"No Description\" src=\"https://www.blognone.com/sites/default/files/externals/13122757f649ba0685ad93b796b5cc19.jpg\" /></p>\n</div></div></div><div><div>Topics: </div><div><div><a href=\"/topics/chatgpt\">ChatGPT</a></div><div><a href=\"/topics/openai\">OpenAI</a></div><div><a href=\"/topics/service-outage\">Service Outage</a></div></div></div>","author":"arjin","siteTitle":"Blognone","siteHash":"ededadcf18490b3937e7dd89ebe8c00dc129addbdf1ebe4aff1f458146693da0","entryHash":"b033954a8acf6710ac0deec8675c6fa04fb67123a5d09df94a360875919daee3","category":"Thai"}